hamocc/mo_param_bgc.F90

```diff
@@ -176,34 +347,34 @@ subroutine ini_fields_atm(kpie,kpje)
   end subroutine
 
   !---------------------------------------------------------------------------------------------------------------------------------
-  subroutine ini_stoichiometry()
+ ! subroutine ini_stoichiometry()
     !
     ! Initialize FIXED stoichiometric parameters for iHAMOCC
     !
     ! extended redfield ratio declaration
     ! Note: stoichiometric ratios are based on Takahashi etal. (1985)
     ! P:N:C:-O2 + 1:16:122:172
-    ro2ut  = 172.
-    rcar   = 122.
-    rnit   = 16.
-    rnoi   = 1./rnit
-    riron  = 5.*rcar*1.e-6       ! fe to P ratio in organic matter
+!    ro2ut  = 172.
+!    rcar   = 122.
+!    rnit   = 16.
+!    rnoi   = 1./rnit
+!    riron  = 5.*rcar*1.e-6       ! fe to P ratio in organic matter
 
     ! stoichiometric ratios for denitrification from Paulmier et al. 2009, Table 1 and
     ! equation 18. Note that their R_0=ro2ut-2*rnit.
-    rdnit0 = 0.8*ro2ut           ! moles nitrate lost for remineralisation of 1 mole P
-    rdnit1 = 0.8*ro2ut - rnit    ! moles nitrate net  for remineralisation of 1 mole P
-    rdnit2 = 0.4*ro2ut           ! moles N2 released  for remineralisation of 1 mole P
+!    rdnit0 = 0.8*ro2ut           ! moles nitrate lost for remineralisation of 1 mole P
+!    rdnit1 = 0.8*ro2ut - rnit    ! moles nitrate net  for remineralisation of 1 mole P
+!    rdnit2 = 0.4*ro2ut           ! moles N2 released  for remineralisation of 1 mole P
 
     ! stoichiometric ratios for N2O loss by "intermediate dinitrification". Note that there
-    ! is no nitrate created by this process, organic N is released as N2
-    rdn2o1 = 2*ro2ut - 2.5*rnit  ! moles N2O used for remineralisation of 1 mole P
-    rdn2o2 = 2*ro2ut - 2*rnit    ! moles N2 released  for remineralisation of 1 mole P
+!    ! is no nitrate created by this process, organic N is released as N2
+!    rdn2o1 = 2*ro2ut - 2.5*rnit  ! moles N2O used for remineralisation of 1 mole P
+!    rdn2o2 = 2*ro2ut - 2*rnit    ! moles N2 released  for remineralisation of 1 mole P
 
     !ik for interaction with sediment module
-    o2ut   = 172.
-    rno3   = 16.
-  end subroutine
+!    o2ut   = 172.
+!    rno3   = 16.
+!  end subroutine
 
```

Comment:
I guess this whole routine will be removed eventually?

Reply:
Yes, done.